### PR TITLE
CONSTRUCT3D Construct 1 & Construct 1 XL print profiles

### PR DESCRIPTION
**Pull request by Construct3D-AM**
Additions include:

- CONSTRUCT3D.json overview for populating printer selection menu
- required pictures
- Build plate models
- 2 machine files and 2 nozzle files
- 3 filament presets
- 8 print profiles (4 C1,  4 C1XL)

Note:

- Some error logs have cropped up meaning the printers will not populate the list. However these errors do not match the visible files so I am pulling a blank on why this is happening. I do not recommend merging until this issue is solved.

### CODE REVIEW

**Comment by Construct3D-AM** [comment]
Updated the files to fix the typo,  Building was successful however the behaviour is the same, a consistent error keeps appearing as follows:
`2024-02-23 00:21:07.261843[Thread 0x0000492c]:Slic3r::ConfigBase::load_from_json: parse C:\Users\Slave-0\AppData\Roaming\OrcaSlicer\system/CONSTRUCT3D/process/0.25mm Industrial@Construct 1.json got a nlohmann::detail::parse_error, reason = [json.exception.parse_error.101] parse error at line 1, column 1: syntax error while parsing value - unexpected end of input; expected '[', '{', or a literal`

This error at Line 1 Collumn 1 also appears in the `"/resources/profiles/CONSTRUCT3D.json"` AND weirdly shows up also in a Prusa file as well.   

I am unsure why the `{` I use is not being recognised as the correct symbol.

**Comment by SoftFever** [comment]
> Updated the files to fix the typo, Building was successful however the behaviour is the same, a consistent error keeps appearing as follows: `2024-02-23 00:21:07.261843[Thread 0x0000492c]:Slic3r::ConfigBase::load_from_json: parse C:\Users\Slave-0\AppData\Roaming\OrcaSlicer\system/CONSTRUCT3D/process/0.25mm Industrial@Construct 1.json got a nlohmann::detail::parse_error, reason = [json.exception.parse_error.101] parse error at line 1, column 1: syntax error while parsing value - unexpected end of input; expected '[', '{', or a literal`
> 
> This error at Line 1 Collumn 1 also appears in the `"/resources/profiles/CONSTRUCT3D.json"` AND weirdly shows up also in a Prusa file as well.
> 
> I am unsure why the `{` I use is not being recognised as the correct symbol.

There tons of other inconsistent names in the profiles.
E.g. 
`0.25mm Industrial @Construct 1` versus `0.25mm Industrial@Construct 1`

Try fixing all these issues first.
You can search key words like "parse_error" in the log file to find json files with errors.

**Comment by Construct3D-AM** [comment]
Hopefully this is the last needed update. The behaviour is still the same with no printers showing up. But on the plus side, the logs dont show any errors any `parse_error`s any more.

**Comment by SoftFever** [comment]
> Hopefully this is the last needed update. The behaviour is still the same with no printers showing up. But on the plus side, the logs dont show any errors any `parse_error`s any more.
> 
> The only error that is remaining is: `2024-02-26 00:44:56.244426[Thread 0x000033bc]:Slic3r::PresetBundle::load_vendor_configs_from_json, got error when parse filament setting from C:\Users\Slave-0\AppData\Roaming\OrcaSlicer\system/CONSTRUCT3D/filament/fdm_filament_common.json`

It just occurs to me that it is probably because the modified files are not copied over to C:\Users\Slave-0\AppData\Roaming\OrcaSlicer\system folder.
Can you try delete the folder?

**Comment by SoftFever** [comment]
Meanwhile, I have created a tool to help validate profiles locally.
Please take a look here: 
https://github.com/SoftFever/Orca_tools/releases/tag/1

**Comment by Construct3D-AM** [comment]
Deleting the folder did the trick. I can confirm the modified files does not transfer over to the roaming folders.

Additionally the validation tool you have created has worked splendidly in finding a non critical error with the nozzle variants which has now been fixed.

Both printers now correctly populate the printers list, and all seems to be working correctly.

File: resources/profiles/CONSTRUCT3D/process/fdm_process_common.json
Comment (SoftFever) [inline resources/profiles/CONSTRUCT3D/process/fdm_process_common.json:1]: Seems like you deleted `resources\profiles\Prusa\process\fdm_process_common.json` file which caused errors when loading Prusa profiles.
Can you revert it?

**Review by SoftFever — CHANGES_REQUESTED** [review]

**Comment by SoftFever** [comment]
> Deleting the folder did the trick. I can confirm the modified files does not transfer over to the roaming folders.
> 
> Additionally the validation tool you have created has worked splendidly in finding a non critical error with the nozzle variants which has now been fixed.
> 
> Both printers now correctly populate the printers list, and all seems to be working correctly.

Glad to hear it works well.
Meanwhile can you take a look at my comments, looks like you deleted a file in Prusa profile hence the CI/CD errored out

**Comment by Construct3D-AM** [comment]
Reversion done.  Looks like instead of a copy paste I accidently did a cut paste. The file has been re-added from current latest build.

**Pull request merged**